sdk/monitor/Azure.Monitor.Ingestion/perf/IngestionClientTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Test.Perf;
+using CommandLine;
+
+namespace Azure.Monitor.Ingestion.Perf
+{
+    public class IngestionClientTest : PerfTest<IngestionClientTest.IngestionClientPerfOptions>
+    {
+        /* please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/perf/TemplateClientTest.cs to write perf test. */
+
+        public IngestionClientTest(IngestionClientPerfOptions options) : base(options)
+        {
+        }
+        public class IngestionClientPerfOptions : PerfOptions
+        {
+        }
+
+        public override void Run(CancellationToken cancellationToken)
+        {
+        }
+
+        public override async Task RunAsync(CancellationToken cancellationToken)
+        {
+            await Task.Run(() =>
+            {
+                Console.WriteLine("exec some async operation");
```

Comment:
Nit - consider `throw new NotImplementedException();` after the `Console.WriteLine` so it's super obvious if anyone tries to run this.

Unless this causes a bunch of noise in the CI...  then ignore the feedback.  :smile: